src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java

```diff
@@ -41,6 +41,7 @@ public class TikuiModuleFactory {
   //@formatter:off
   public JHipsterModule buildModule(JHipsterModuleProperties properties) {
     return moduleBuilder(properties)
+      .preCommitActions(stagedFilesFilter("{src/**/,}*.pug"), preCommitCommands("eslint --fix", "prettier --write"))
```

Comment:
Should be sufficient:
```suggestion
      .preCommitActions(stagedFilesFilter("*.pug"), preCommitCommands("eslint --fix", "prettier --write"))
```

Reply:
yes it works, I update